manifests/d/DearMob/iPhoneManager/6.0/DearMob.iPhoneManager.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: DearMob.iPhoneManager
+PackageVersion: 6.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft

```

Comment:
The installer exits with exit code 1223 when the manifest is run. Custom installer switches will need to be provided